pubspec.yaml

```diff
@@ -56,6 +56,7 @@ dependencies:
   freezed_annotation: ^2.2.0
   json_annotation: ^4.7.0
   memoize: ^3.0.0
+  home_widget: ^0.6.0
```

Comment:
bump it to 0.7.0

Reply:
Will do. I saw that they now [recommend a new approach using Glance and Jetpack compose](https://docs.page/ABausG/home_widget/setup/android) instead of XML files. I'll do that in a followup PR.

Reply:
Nice!

Reply:
Updated.